cap-ci/pipeline.template

```diff
@@ -1139,6 +1147,54 @@ jobs:
             {{- print $import_aks }}
             {{- end }}
             {{- print $stratos }}
+{{ end }}
+
+{{- if index $config.jobs "clean-kubecf" }}
```

Comment:
I don't this this should be conditional.

Reply:
changed already.